.env_template

```diff
@@ -3,3 +3,7 @@ export CONTAINER_DOMAIN=     # dns.podman or docker.internal
 export VAULT_LICENSE= # retrieve from https://www.hashicorp.com/products/vault/trial
 IPA_REALM=            # DNS.PODMAN or DOCKER.INTERNAL
 
+# Elasticsearch settings
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html
+STACK_VERSION=8.4.1   # Version of Elastic products
+MEM_LIMIT=536870912   # 512M, increase or decrease based on the available host memory (in bytes)
```

Comment:
I tried to run it w/o any limits first (minimal config), but Elastic and Kibana decided to get all they can..